### PR TITLE
Chores form

### PR DESCRIPTION
**Pull request by mboneil10**
Besides the fact that I don't have an actual URL on the reminders page, I don't understand what else I'm missing.

### CODE REVIEW

**Comment by mboneil10** [comment]
Oh, and I'll also add deleting as a possibility

File: cleanslate/doma/templates/chore_edit_form.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "base_generic.html" %}
```
Comment (L4r1k) [inline cleanslate/doma/templates/chore_edit_form.html]: I think this should be 
`{% extends "base.html" %}`

File: cleanslate/doma/urls.py [hunk 2]
```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
+    url(r'^chore/(?P<pk>[-\w]+)/edit/$', views.edit_chore_deadline, name = 'edit-chore-deadline'),
```
Comment (Cobchise) [inline cleanslate/doma/urls.py]: I'm going to do more reading on this but I'm wondering if the ID comes after edit as I've found an example like this:
`r'^book/edit/(?P<object_id>\d+)/$', create_update.update_object, book_info)`
they also made a big deal about the \d+ at the end

**Review by onocy — DISMISSED** [review]

**Comment by mboneil10** [comment]
My newest issue is that I was able to use the form, but made no changes to the chore in question. Gah why.

**Comment by mboneil10** [comment]
OH WAIT NO IT WORKS

**Comment by mboneil10** [comment]
Now I need to make sure the edit link is connected to the chore form. As well as add capability for adding and deleting chores.

**Comment by mboneil10** [comment]
Closing until finished

**Pull request closed without merging**